packages/styles/icon-button.css

```diff
@@ -61,11 +61,13 @@
 /* Ensure both <a> and <button> IconButtons are aligned correctly. See #846. */
 a.IconButton {
   vertical-align: middle;
+  line-height: unset;
 }
 
 .IconButton .Icon {
-  width: 100%;
-  height: 100%;
+  height: var(--space-small);
+  width: var(--space-small);
+  margin-top: 2px;
```

Comment:
Can you help me understand the motivation behind this change? I'm also wondering if this will mess up anyone's usage of IconButton if they have overridden the IconButton's height and width.

Reply:
@bobbyomari said the icon button's SVGs are too big. He wanted them to be this size.

Reply:
Instead of setting the constraints on the Icon, can we solve this by adding padding to the IconButton container?

Reply:
I agree this change might cause issues as @thuey is suggesting. Also, if we want them to be a fixed size, we should instead be using something like `align-self: center` to let the browser do the centering instead approximating it with margins.

Reply:
That's a good idea, I'll experiment with that.